src/core/Checkbox/Checkbox.tsx

```diff
@@ -95,6 +102,7 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
             'iui-disabled': disabled,
             [`iui-${status}`]: !!status,
             'iui-loading': isLoading,
+            'iui-checkbox-visibility': variant === 'eyeball',
```

Comment:
> I was thinking having a prop instead, where user can provide his own SVGs for every state of checkbox rather than having a specific variant in the checkbox component itself.

@veekeys Not sure how to do this, I think we do need to know that it is going to be a visibility checkbox to set this class name

Reply:
I think we discussed this several times before. You can't simply pass any SVGs as we are applying classes to the inside elements of SVG. Maybe we can discuss this during our Friday meeting.